benchmarks/benchmarks/sparse_linalg_svds.py

```diff
@@ -1,31 +1,68 @@
 import os
+import time
+import warnings
 import numpy as np
 from .common import Benchmark, safe_import
 
 with safe_import():
+    from scipy.linalg import svd
     from scipy.sparse.linalg import svds
 
+# ensure that we are benchmarking a consistent outcome;
+# (e.g. if the code wasn't able to find a solution accurately
+# enough the timing of the benchmark would become useless).
```

Comment:
This comment is now free-floating. AFAICT, it should be attached above the `if accuracy < self.tol:`